CONTRIBUTORS.txt

```diff
@@ -152,11 +164,20 @@ needs to be listed here.
 - Volodymyr Sapsai <vsapsai@users.noreply.github.com>
 - Will Lisac <will@lisac.org>
 - Wilson Ding <hello@wilsonding.com>
+- YR Chen <stevapple@icloud.com>
+- Yim Lee <yim_lee@apple.com>
 - akash-55 <61596874+akash-55@users.noreply.github.com>
+- buttaface <butta@fastem.com>
+- carolinacass <67160898+carolinacass@users.noreply.github.com>
+- dkz2 <132388278+dkz2@users.noreply.github.com>
 - fadi-botros <botros_fadi@yahoo.com>
 - jemmons <jemmons@users.noreply.github.com>
 - pokryfka <pokryfka@gmail.com>
+- ser <122270051+ser-0xff@users.noreply.github.com>
```

Comment:
CC @ser-0xff  if you want a real email address here, please submit a pull request with an appropriate entry to the `.mailmap` file. You'll find plenty of examples there, but essentially if you add

```
Your Name <your-real@email.address> <123456+the-weird-github-one@users.noreply.github.com>
```

and then run `scripts/generate_contributors_list.sh` you should appear with proper name & email.